ocaml/xapi/create_storage.ml

```diff
@@ -42,14 +42,13 @@ let maybe_reenable_cluster_host __context =
   let host = Helpers.get_localhost __context in
   match Xapi_clustering.find_cluster_host ~__context ~host with
   | Some self ->
-      if not (Db.Cluster_host.get_enabled ~__context ~self) then
-        Xapi_cluster_host.enable ~__context ~self
+     Xapi_cluster_host.enable ~__context ~self
   | None -> ()
 
 let plug_unplugged_pbds __context =
   (* If the plug is to succeed for SM's requiring a cluster stack
    * we have to enable the cluster stack too if we have one *)
-  maybe_reenable_cluster_host __context;
+  log_and_ignore_exn(fun () -> maybe_reenable_cluster_host __context);
```

Comment:
So now we will just log an error below if we failed to enable clustering, instead of failing the whole function with an exception. Maybe that's better indeed, as it's closer to the original behaviour 🤔

Reply:
Yes, repair worked the last time I checked. It does call pool resync.